packages/web-app/src/modules/storefront-views/blocks/StorefrontHeroBlock.tsx

```diff
@@ -0,0 +1,38 @@
+import { rewardRoute } from '../../../RouteUtils'
+import { StorefrontHeroBlockProps } from '../../storefront/models'
+import { StorefrontHero } from '../components/StorefrontHero'
+import { StorefrontHeroItem } from '../components/StorefrontHeroItem'
+
+interface Props {
+  block: StorefrontHeroBlockProps
+}
+
+export const StorefrontHeroBlock = ({ block }: Props) => {
+  return (
+    <StorefrontHero key={block.id} title={block.title}>
+      {block.items.map((hero) => {
+        const heading = hero.reward?.name || hero.heading
```

Comment:
These are flipped... We want to use the explicitly defined value first, and we want to _fallback_ to the reward (if defined).

```suggestion
        const heading = hero.heading || hero.reward?.name
```